### PR TITLE
add titan raw case

### PR DESCRIPTION
**Pull request by Connor1996**
### What problem does this PR solve? 

Add titan raw case test 

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: tests/titan/titan.go [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+			select {
+			case <-ctx.Done():
+				log.Info("Test finished.")
+				return nil
+			default:
+			}
+
+			start = []byte(fmt.Sprintf("v%3d", i))
+			end = []byte(fmt.Sprintf("%v3d", i+1))
```
Comment (cosven) [inline tests/titan/titan.go:153]: `v%3d`

File: tests/titan/titan.go [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+				}
+				err := c.cli.Put(ctx, []byte(key), val)
+				if err != nil {
+					log.Fatalf("put key %v value %v failed: %v", key, val, err)
+				}
+
+				if j != 0 && j%1000 == 0 {
+					for k := j - 1000; k < j; k++ {
+						if k%5 == 0 {
+							err = c.cli.Delete(ctx, []byte(key))
```
Comment (cosven) [inline tests/titan/titan.go:109]: when j == 1000 and k == 5, the key is 1000 instead of 5

File: tests/titan/titan.go [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+func (c *titanClient) NextRequest() interface{} {
+	panic("implement me")
+}
+
+func (c *titanClient) DumpState(ctx context.Context) (interface{}, error) {
+	panic("implement me")
+}
+
+func (c *titanClient) Start(ctx context.Context, cfg interface{}, clientNodes []types.ClientNode) error {
+	for {
```
Comment (cosven) [inline tests/titan/titan.go:77]: maybe we should delete range `(v  0, v101)` before test starts

File: tests/titan/titan.go [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+		for i := 1; i <= 100; i++ {
+			rnd := rand.New(rand.NewSource(time.Now().Unix()))
+			keyHashMap := make(map[string]uint32)
+			hashs := make(map[uint32]struct{})
+
+			for j := 0; j <= 40000; j++ {
+				key := fmt.Sprintf("v%3dj%5d", i, j)
```
Comment (cosven) [inline tests/titan/titan.go:84]: `"v%3dj%5d"` will produce string like `v  1j     1`, which is padded with whitespace, I think string padding with zero is more readable.

**Comment by cosven** [comment]
LGTM

**Comment by mahjonp** [comment]
> LGTM

Thanks, cosven. I have granted you the review privilege.

**Pull request merged**